test/known_issues/test-stdout-buffer-flush-on-exit.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/2148
+require('../common');
+const assert = require('assert');
+const execSync = require('child_process').execSync;
+
+const longLine = 'foo bar baz quux quuz aaa bbb ccc'.repeat(80);
+const expectedLength = (longLine.length * 999) + 1;
```

Comment:
Yeah, and we may want to compare the output instead of comparing the lengths just to be extra sure we get what we expect.